maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationService.java

```diff
@@ -28,24 +33,26 @@
 @Service
 @RequiredArgsConstructor
 public class MeansAssessmentOrchestrationService {
-
     private final ContributionService contributionService;
     private final ProceedingsService proceedingsService;
     private final MeansAssessmentService meansAssessmentService;
     private final MaatCourtDataService maatCourtDataService;
     private final AssessmentSummaryService assessmentSummaryService;
     private final FeatureDecisionService featureDecisionService;
+    private final WorkflowPreProcessorService workflowPreProcessorService;
+    private final MeansAssessmentMapper meansAssessmentMapper;
 
     public FinancialAssessmentDTO find(int assessmentId, int applicantId) {
         return meansAssessmentService.find(assessmentId, applicantId);
     }
 
     public ApplicationDTO create(WorkflowRequest request) {
-
         ApplicationDTO application = request.getApplicationDTO();
         try {
             Long repId = application.getRepId();
             log.debug("Creating Means assessment for applicationId = " + repId);
+
+            preProcessRequest(request, Action.CREATE_ASSESSMENT);
```

Comment:
Is this the correct action? There wasn't anything obvious in the stored procedure.

Reply:
this is the correct action, the stored procedures have been retired.